src/Umbraco.Web.UI.Client/src/views/propertyeditors/listview/listview.controller.js

```diff
@@ -59,6 +59,37 @@ function listViewController($rootScope, $scope, $routeParams, $injector, $cookie
         items: []
     };
 
+        
+    $scope.currentNodePermissions = {}
+    
+    //Just ensure we do have an editorState
+    if(editorState.current){
+        
+        //Get Current User to check if their usertype is an admin (overwrites all permissions!)
+        userService.getCurrentUser().then(function(data){
+            
+            //Fetch current node allowed actions for the current user
+            //This is the current node & not each individual child node in the list
+            var currentUserPermissions = editorState.current.allowedActions;
+                
+            //Create a nicer model rather than the funky & hard to remember permissions strings
+            $scope.currentNodePermissions = {
+                "isAdminUser": data.userType.toLowerCase() === "admin",

```

Comment:
We shouldn't really do this, it doesn't matter if the user is an 'admin' user type, a user type is simply a way to flag default permissions, their permissions could be changed by another admin